src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -370,4 +370,28 @@ public function getBytesChanged(Event $event, string $dbName, array $pageIds, ar
 
         return (int)$res->fetchColumn();
     }
+
+    /**
+     * Get the list of users participating in an event with no predefined user list
+     *
+     * @param string $dbName
+     * @param int[] $pageIds
+     * @param Event $event
+     * @return string[]
+     */
+    public function getUsersFromPageIDs(string $dbName, array $pageIds, Event $event): array
+    {
+        $revisionTable = $this->getTableName('revision');
+        $rqb = $this->getReplicaConnection()->createQueryBuilder();
+        $rqb->select('DISTINCT(rev_user_text)')
+            ->from("$dbName.$revisionTable")
+            ->where('rev_page IN (:pageIds)')
+            ->andWhere('rev_user <> 0')
```

Comment:
I didn't even know about the `<>` operator! I always used `!=`. Anyway, a tiny nitpick -- it may be a little misleading here because accounts will always have IDs greater than zero, and we don't want NULL values (though I realize none exist).